### PR TITLE
Feature/four of a kind

### PR DESCRIPTION
**Pull request by yodayoda1**

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -155,4 +170,16 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
```
Comment (crottsl7324) [inline source/CardGames.py]: what did you change here

Comment (yodayoda1) [reply source/CardGames.py]: I was adding a line that would call the shuffle feature from above on to the deck feature.

Comment (davidcloar) [reply source/CardGames.py]: Where is four of a kind?

**Review by davidcloar — APPROVED** [review]

File: source/CardGames.py (hunk 1 above)
Comment (yodayoda1) [reply source/CardGames.py]: it is at the bottom of my committed code, under the Game class.

**Review by crottsl7324 — COMMENTED** [review]
Nice job fixing the problem.

**Review by crottsl7324 — APPROVED** [review]

**Pull request merged**